sway/tree/output.c

```diff
@@ -60,6 +60,13 @@ static void restore_workspaces(struct sway_output *output) {
 		struct sway_workspace *ws = root->noop_output->workspaces->items[0];
 		workspace_detach(ws);
 		output_add_workspace(output, ws);
+		for (int i = 0; i < ws->floating->length; i++) {
```

Comment:
Is it possible to add a comment here to explain in which situation we need to check for `width == 0 || height == 0 || …`?

Reply:
Done